test/gen_lsp_test.exs

```diff
@@ -146,4 +146,12 @@ defmodule GenLSPTest do
       "message" => "Method Not Found"
     })
   end
+
+  test "can start multiple servers in one test/2" do
+    assert %{} = server(GenLSPTest.ExampleServer, name: :example_server_2)
+
+    assert_raise RuntimeError, ~r/more than one child specification has the id: :example_server_2buffer/, fn ->
+      server(GenLSPTest.ExampleServer, name: :example_server_2)
+    end
+  end
```

Comment:
this test is testing that you can't open 2 with the same name, not that it can start multiple

Reply:
it already started a `server` in the setup

Reply:
oh, missed that, i don't think you need the assert raise assertion

i would probably assert on if the server is alive then with the `alive?` function.

Reply:
addressed